runtime-extension-model/src/main/java/module-info.java

```diff
@@ -56,12 +53,11 @@
 
   exports org.mule.runtime.config.internal.dsl.processor.xml.provider to
       org.mule.runtime.extensions.mule.support;
-  exports org.mule.runtime.config.internal.error to
-      org.mule.runtime.core,
-      org.mule.runtime.extensions.mule.support,
-      org.mule.runtime.spring.config,
-      org.mule.runtime.artifact.ast.serialization.test;
-  
+
+  provides org.mule.runtime.core.api.extension.provider.RuntimeExtensionModelProvider with
+      org.mule.runtime.core.api.extension.provider.CoreRuntimeExtensionModelProvider,
+      org.mule.runtime.core.api.extension.provider.OperationDslExtensionModelProvider;
```

Comment:
`OperationDslExtensionModelProvider` belongs to `extensions-mule-support`, not here